src/main/java/duke/taskmanager/Event.java

```diff
@@ -0,0 +1,78 @@
+package duke.taskmanager;
+
+import duke.exceptions.emptyDescException;
+import duke.exceptions.unrecogException;
+import duke.exceptions.unspecTimeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+/**
+ * Event class of parent class Tasks
+ */
+public class Event extends Tasks{
+    String icon = "[E]";
+    String desc;
+    String timeframe;
+    String time1, time2;
+    LocalDate date1, date2;
```

Comment:
would it be better to add some kind of access modifiers to these variables? similar to above